indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -1,6 +1,38 @@
 {% extends 'events/surveys/management/_survey_base.html' %}
 
 {% block content %}
+    {% if survey.private and survey.state.name in ('active_and_clean', 'active_and_answered') %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-lock"></div>
+                <div class="text">
+                    <div class="label">
+                        {%- trans %}The survey is private{% endtrans -%}
+                    </div>
+                    {% trans -%}
+                        The survey is accessible only by people with this
+                    {%- endtrans %}
+                    <a href="{{ url_for('.display_survey_form', survey, token=survey.uuid, _external=True) }}"
+                       class="js-copy-to-clipboard"
+                       title="{% trans %}Copy link to clipboard{% endtrans %}"
+                       data-clipboard-text="{{ url_for('.display_survey_form', survey, token=survey.uuid, _external=True) }}"
+                       data-clipboard-action="copy">
+                        {%- trans %}link{% endtrans -%}
```

Comment:
That's not a good way of doing i18n. Never build a sentence that spans multiple trans tags - otherwise you basically assume that "link" is the last word.

 Check [here](https://github.com/indico/indico/blob/devel/indico/modules/events/management/templates/_management_frame.html#L8-L11) for a way of doing it better